.upgrade.yml

```diff
@@ -12,20 +12,26 @@ mappings:
   DataQuery: SilverStripe\ORM\DataQuery
   DataQuery_SubGroup: SilverStripe\ORM\DataQuery_SubGroup
   DB: SilverStripe\ORM\DB
-  SS_Filterable: SilverStripe\ORM\SS_Filterable
+  SS_Filterable: SilverStripe\ORM\Filterable
+  SilverStripe\ORM\SS_Filterable: SilverStripe\ORM\Filterable
   GroupedList: SilverStripe\ORM\GroupedList
   HasManyList: SilverStripe\ORM\HasManyList
   HiddenClass: SilverStripe\ORM\HiddenClass
-  SS_Limitable: SilverStripe\ORM\SS_Limitable
+  SS_Limitable: SilverStripe\ORM\Limitable
+  SilverStripe\ORM\SS_Limitable: SilverStripe\ORM\Limitable
   SS_List: SilverStripe\ORM\SS_List
-  SS_ListDecorator: SilverStripe\ORM\SS_ListDecorator
+  SS_ListDecorator: SilverStripe\ORM\ListDecorator
+  SilverStripe\ORM\SS_ListDecorator: SilverStripe\ORM\ListDecorator
   ManyManyList: SilverStripe\ORM\ManyManyList
-  SS_Map: SilverStripe\ORM\SS_Map
-  SS_Map_Iterator: SilverStripe\ORM\SS_Map_Iterator
+  SS_Map: SilverStripe\ORM\Map
```

Comment:
I think there's a non-trivial risk that PHP8 reserves `map` as a keyword. Let's leave this one as `SS_Map`